packages/dialtone-vue2/components/tooltip/tooltip.stories.js

```diff
@@ -66,6 +66,12 @@ export const argTypesData = {
     },
   },
 
+  appendTo: {
+    control: {
+      type: null,
```

Comment:
Why null? Don't we want to test this in storybook?

Reply:
yeah because it's non-reactive changing it live in storybook will do nothing.

Reply:
ooh I see